digiwf-email-integration/src/main/java/io/muenchendigital/digiwf/email/integration/domain/model/Attachment.java

```diff
@@ -0,0 +1,38 @@
+package io.muenchendigital.digiwf.email.integration.domain.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+
+/**
+ * Attachment File you want to get from the S3 storage.
+ */
+@Data
+@RequiredArgsConstructor
+@AllArgsConstructor
+public class Attachment {
```

Comment:
General comment: Attribute validation should not be included within the model/dto/entity class. Better is to write in own validation class.

Reply:
Removed methods